form/src/main/java/org/dhis2/form/ui/provider/inputfield/FieldProvider.kt

```diff
@@ -75,7 +78,7 @@ internal fun FieldProvider(
                 }
             }
         }
-
+    val focusManager = LocalFocusManager.current
```

Comment:
ok, this interesting. It is a good idea to allow the Form release the focus sometimes, also when it is the last item etc... so I will suggest you to declare it one level up, instead of declaring it in FieldProvider you can do it in Form.kt and pass it as parameter. If not, we can do it in the future, in other issue.

Reply:
Good idea! I have already updated it